modules/swagger-codegen/src/main/resources/ruby/api.mustache

```diff
@@ -70,9 +70,10 @@ module {{moduleName}}
 
       # http body (model)
       {{^bodyParam}}post_body = nil
-      {{/bodyParam}}{{#bodyParam}}post_body = @api_client.object_to_http_body({{#required}}{{{paramName}}}{{/required}}{{^required}}opts[:'{{{paramName}}}']{{/required}})
-      {{/bodyParam}}
-
+      {{/bodyParam}}{{#bodyParam}}{{#isBinary}}post_body = {{#required}}{{{paramName}}}{{/required}}{{^required}}opts[:'{{{paramName}}}']{{/required}}
+      post_body = post_body.pack('C*') if post_body
```

Comment:
Nearly the same comment from https://github.com/swagger-api/swagger-codegen/pull/1891/files#diff-630a0645a0b3fabb89412db600734cfbR245

A array of 8bit ints is a very non-standard data format in Ruby. As a user, I would prefer `post_body` here without processing.